config/jobs/kubernetes/sig-release/release-branch-jobs/1.15.yaml

```diff
@@ -302,7 +302,7 @@ periodics:
 - annotations:
     fork-per-release-generic-suffix: "true"
     fork-per-release-periodic-interval: 2h 6h 24h
-    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com
+    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, osp-engprod@google.com
```

Comment:
maybe something like osp-engprod+oss-alerts@google.com, to help filtering

Reply:
setup a new k8-infra-oncall@ group instead